bin/cheribsdtest/cheribsdtest_bounds_stack.c

```diff
@@ -72,13 +72,11 @@ test_bounds_precise(void * __capability c, size_t expected_len)
 #ifdef __CHERI_PURE_CAPABILITY__
 	/* Confirm precise lower bound: offset of zero. */
 	CHERIBSDTEST_VERIFY2(offset == 0,
-	    "offset (%jd) not zero: " _CHERI_PRINTF_CAP_FMT, offset,
-	    _CHERI_PRINTF_CAP_ARG(c));
+	    "offset (%jd) not zero: %#.16lp", offset, c);
```

Comment:
I suspect here and in many places, we would actually be fine with just `%#lp` without needing the padded address.  Probably only places where we print a table or list of values do we really need the `.16` precision.